frontend/public/components/data-view-poc/DataViewPodList.tsx

```diff
@@ -0,0 +1,801 @@
+      },
+      [podColumnInfo.status.id]: {
+        id: podColumnInfo.status.id,
+        props: { className: podColumnInfo.status.classes },
+        cell: <PodStatus pod={pod} />,
+      },
+      [podColumnInfo.ready.id]: {
+        id: podColumnInfo.ready.id,
+        props: { className: podColumnInfo.ready.classes },
+        cell: `${readyCount}/${totalContainers}`,
+      },
+      [podColumnInfo.restarts.id]: {
+        id: podColumnInfo.restarts.id,
+        props: { className: podColumnInfo.restarts.classes },
+        cell: restarts,
+      },
+      [podColumnInfo.owner.id]: {
+        id: podColumnInfo.owner.id,
+        props: { className: podColumnInfo.owner.classes },
+        cell: showNodes ? (
+          <ResourceLink kind="Node" name={pod.spec.nodeName} namespace={namespace} />
+        ) : (
+          <OwnerReferences resource={pod} />
+        ),
+      },
+      [podColumnInfo.memory.id]: {
+        id: podColumnInfo.memory.id,
+        props: { className: podColumnInfo.memory.classes },
+        cell: bytes ? `${formatBytesAsMiB(bytes)} MiB` : '-',
+      },
+      [podColumnInfo.cpu.id]: {
+        id: podColumnInfo.cpu.id,
+        props: { className: podColumnInfo.cpu.classes },
+        cell: cores ? t('public~{{numCores}} cores', { numCores: formatCores(cores) }) : '-',
+      },
+      [podColumnInfo.created.id]: {
+        id: podColumnInfo.created.id,
+        props: { className: podColumnInfo.created.classes },
+        cell: <Timestamp timestamp={creationTimestamp} />,
+      },
+      [podColumnInfo.node.id]: {
+        id: podColumnInfo.node.id,
+        props: { className: podColumnInfo.node.classes },
+        cell: <ResourceLink kind="Node" name={pod.spec.nodeName} namespace={namespace} />,
+      },
+      [podColumnInfo.labels.id]: {
+        id: podColumnInfo.labels.id,
+        props: { className: podColumnInfo.labels.classes },
+        cell: <LabelList kind={kind} labels={labels} />,
+      },
+      [podColumnInfo.ipaddress.id]: {
+        id: podColumnInfo.ipaddress.id,
+        props: { className: podColumnInfo.ipaddress.classes },
+        cell: pod?.status?.podIP ?? '-',
+      },
+      [podColumnInfo.traffic.id]: {
+        id: podColumnInfo.traffic.id,
+        props: { className: podColumnInfo.traffic.classes },
+        cell: <PodTraffic podName={name} namespace={namespace} />,
+      },
+    };
+    const dataViewRow: DataViewTd[] = columns.map(({ id }) => rowCells[id]);
+    const actionsRow: DataViewTd = {
+      id: '',
+      props: { className: Kebab.columnClass },
+      cell: <LazyActionMenu context={context} isDisabled={phase === 'Terminating'} />,
+    };
+    // Always add the actions column
+    dataViewRow.push(actionsRow);
+    return dataViewRow;
+  });
+}
+
+/**
+ * The sorting logic was copied over from public/components/factory/Table/VirtualizedTable.tsx file
+ * For the most part it is 1:1
+ * column removal based on screen size was omitted from POC
```

Comment:
Do we want to preserve column removal as that is not PatternFly default behavior.  @spadgett, @jhadvig, WDYT?

Reply:
And if we do go the sticky column route, we could conceivably drop the maximum number of columns (currently 9) that can be active when column management is in use?  

Also wondering if column management shouldn't be enabled on all tables?  We've had requests for columns to be resizable, but PatternFly doesn't yet support that.  Being able to manage the columns may help in that regard?

Reply:
You can still use the original column "hiding" mechanism. You have to use different elements as your width baseline.

About the sticky columns, you can use any valid PF prop on your columns/cells.

Reply:
> You can still use the original column "hiding" mechanism. You have to use different elements as your width baseline.

Are you referring to column removal from right to left as the viewport shrinks?  If so, can you clarify what you mean by "use different elements as your width baseline"?

Reply:
Yes. I mean that. You can still do that, Its all about what data you feed to the data view. If you used the virtualized table as your element where you computed the widht and visible elements, now you would do that with the data view columns instead.